app/assets/javascripts/file-input.js

```diff
@@ -0,0 +1,13 @@
+$(function () {
+  $('.js-file-input input').on('change', function (e) {
+    var file = e.target.files[0];
+    var fileReader = new FileReader();
+    fileReader.onload = function () {
+      var dataUri = this.result;
+      $('.js-file-input__preview img').remove();
+      $('.js-file-input__preview p').text("画像を変更");
+      $('.js-file-input__preview').append('<img src=' + dataUri + '>');
```

Comment:
```suggestion
      $('.js-file-input__preview').append($('img').attr('src', dataUri));
```

の方が安全です。XSS的に 👮 

HTMLを文字列で組み立てないようにするというのが一つのtips(気付きポイント)です。

Reply:
@bluerabbit 
なるほど、ありがとうございます！